unixfs/README.md

```diff
@@ -37,6 +37,16 @@ message Data {
 
 	optional uint64 hashType = 5;
 	optional uint64 fanout = 6;
+	
+	/* file metadata */
+	optional uint32 mtime = 7;
+	optional uint32 atime = 8;
+	optional uint32 ctime = 9;
+	
+	/* file permissions */
+	optional uint32 permissions = 10;
+	optional uint32 uid = 11;
+	optional uint32 gid = 12;
```

Comment:
For now, we can probably stick with modification times and permissions. We probably don't need uid/gid.

Eventually, we may want user _names_ and group _names_ but I don't believe that's a pressing issue.

@andrew?

Reply:
I cut them for now but I’d like @andrew to verify they aren’t needed before merging.